recipe/patches/0006-Vendor-grpc-1.59.3.patch

```diff
@@ -0,0 +1,60 @@
+From 744d936f573f55e6c8568b19b6ecd6ed891abc31 Mon Sep 17 00:00:00 2001
+From: Austin Morton <austin.morton@aquatic.com>
+Date: Fri, 19 Jan 2024 20:11:02 +0000
+Subject: [PATCH] Vendor grpc 1.59.3
+
+---
+ bazel/ray_deps_setup.bzl | 19 +++++++++----------
+ 1 file changed, 9 insertions(+), 10 deletions(-)
+
+diff --git a/bazel/ray_deps_setup.bzl b/bazel/ray_deps_setup.bzl
+index c406ba9281..ae7ee4540a 100644
+--- a/bazel/ray_deps_setup.bzl
++++ b/bazel/ray_deps_setup.bzl
+@@ -89,11 +89,10 @@ def ray_deps_setup():
+     # This is copied from grpc's bazel/grpc_deps.bzl
```

Comment:
Probably should update this comment. Also: please add the version of protobuf (update line 23 rather than removing it)

Reply:
I think it's still copied from that file, just v1.59.3
https://github.com/grpc/grpc/pull/34330/files